docs/features/feature-guides/compliance-forms/complete-a-form.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Complete a Form
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# Complete a DataHub Compliance Form
+<FeatureAvailability/>
+
+This guide provides an example of completing a Compliance Form, covering:
+
+1. Accessing a Form from an Asset Page or the DataHub Cloud Task Center
```

Comment:
You can drop DataHub Cloud and just say "Task Center" 

Then in the description you can say that Task Center (only available in DataHub Cloud)